CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 0.13.3 (Unreleased)
+
+SECURITY:
+ - `tfinstall`: The HashiCorp PGP signing key has been rotated (HCSEC-2021-12). This key is used to verify downloaded versions of Terraform. We recommend all users of terraform-exec upgrade to v0.13.3 for this security fix. [GH-166]
```

Comment:
Shall we also link to the bulletin from here and add something like

> Older versions of Terraform are temporarily retaining the old (now revoked) key, but that key will be removed. Installation via older `terraform-exec` versions will therefore stop working soon and you should upgrade as soon as possible to avoid any interruption.

Reply:
Excellent idea, thank you. Added an extra note and the link.